src/date-picker/date-picker-dialog.jsx

```diff
@@ -113,6 +113,7 @@ const DatePickerDialog = React.createClass({
       root: {
         fontSize: 14,
         color: calendarTextColor,
+        userSelect: 'none; -webkit-user-select: none; -moz-user-select: none; -khtml-user-select: none; -ms-user-select: none',
```

Comment:
`userSelect: 'none',` should be enough, the autoprefixer will do the job.
Could you move this to the `Calendar` component?